### ISSUE TITLE
Question: how is the performance compared to germansentiment?

### ISSUE THREAD

**Issue by Iornoq** [text-analytics-20/news-sentiment-development#2]
Hello,

I was wondering if you've compared the fine-tuned model performance (accuracy) to the germansentiment ([german-sentiment-bert](https://huggingface.co/oliverguhr/german-sentiment-bert)). Sorry if you've listed it somewhere, I was unable to find this data after a quick look around.

Thanks in advance!

**Comment by mdraw** [comment]
Hi, I have now uploaded our project report here: https://github.com/text-analytics-20/news-sentiment-development/blob/main/news-sentiment-report-public.pdf
You can find a quantitative comparison in table 4 on page 17. "BERT generic" refers to the original german-sentiment-bert model and "BERT finetuned" is our model.
